modules/ROOT/pages/index.adoc

```diff
@@ -20,6 +20,66 @@ With Mule you can:
 * Use error constructs and try scopes for rapid debugging.
 * Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
 
+[[install_deploy_manage]]
+== Installation, Deployment, Management
+
+//TODO? THIS BEGS FOR A DIAGRAM TO SHOW THE RELATIONSHIPS DESCRIBED HERE
+
+For a Mule app to run, it must be deployed into an environment where the Mule
+is installed.
+
+Mule is packaged with the xref:7.3@studio::index.adoc[Studio] IDE and with
```

Comment:
Not sure (English ignorance): with or within?

Reply:
"in" or "within" are ok, but I'll change to "within" in the hope that it makes sense to more people